metadata-ingestion/examples/recipes/thrift_to_console.yml

```diff
@@ -0,0 +1,10 @@
+---
+# see https://datahubproject.io/docs/metadata-ingestion/source_docs/file for complete documentation
+source:
+  type: "thrift"
+  config:
+    filename: "./examples/thrift_files/example.thrift"
```

Comment:
Instead of a single file per recipe, can we accept a list of them?

Reply:
This requirement will be resolved in following diffs.